Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -78,11 +78,10 @@ final class RecoveryTests: XCTestCase {
           message: "unexpected code in function"
         ),
       ],
-      applyFixIts: ["insert '>'", "insert newline", "insert expression"],
+      applyFixIts: ["insert '>'", "insert expression"],
```

Comment:
The idea is that the `a` test inserts newline and the `b` test below adds the `;`. So, I think you should be able to remove `testRecovery7b`.

Reply:
Ok, that's it removed now.